.github/workflows/ci.yaml

```diff
@@ -161,6 +164,7 @@ jobs:
   integration:
     name: Test CLN ${{ matrix.name }}
     runs-on: ubuntu-22.04
+    timeout-minutes: 180
```

Comment:
I do not like putting the average minutes, because GitHub is famous to cut Ci performance in some cases and I do not want to end up in a spurious case not related to us.

What do you thin we leave the upper limit for this kind of integration testing and for valgrind?